.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Run game tests
+
+on:
+  push:
+  pull_request:
+    types:
+      - synchronize
+      - opened
+      - ready_for_review
+      - reopened
+  schedule:
+    - cron: 0 0 1 * *
```

Comment:
Do we think it will change? 😄

Reply:
Nope, that was just blindly copied from the other action, whoops.